game/fx/quality_of_life.go

```diff
@@ -66,8 +66,62 @@ func Select(p *match.Player, m *match.Match, containerOwner *match.Player, conta
 	return SelectFilter(p, m, containerOwner, containerName, text, min, max, cancellable, func(x *match.Card) bool { return true })
 }
 
+// Select Draw Count prompts to user to select how many cards to draw with upper and lower limits
+func SelectDrawCount(p *match.Player, m *match.Match, text string, min int, max int) int {
```

Comment:
This can probably be used for other situations than just drawing so maybe we should call it `SelectCount` or `SelectNumber` or something like that?

Reply:
Yea fair point to all. Realized this when I was making the PR but wasn't very keen to changing it anymore at that point. Applied all the changes requested